src/ResourceManager/Network/Commands.Network/ApplicationGateway/SslCertificate/AzureApplicationGatewaySslCertificateBase.cs

```diff
@@ -38,15 +38,14 @@ public class AzureApplicationGatewaySslCertificateBase : NetworkBaseCmdlet
                Mandatory = true,
                HelpMessage = "Certificate password")]
         [ValidateNotNullOrEmpty]
-        [Obsolete("(Get/Set/New)-AzureRmApplicationGatewaySslCertificate: The parameter \"Password\" is being changed from a string to a SecureString in an upcoming breaking change release.")]
-        public string Password { get; set; }
+        public SecureString Password { get; set; }
 
         public PSApplicationGatewaySslCertificate NewObject()
         {
             var sslCertificate = new PSApplicationGatewaySslCertificate();
 
             sslCertificate.Name = this.Name;
-            sslCertificate.Data = Convert.ToBase64String(File.ReadAllBytes(CertificateFile));
+            sslCertificate.Data = Convert.ToBase64String(File.ReadAllBytes(this.CertificateFile));
 #pragma warning disable 0618
```

Comment:
You can get rid of the pragma warnings here because it was suppressing the Obsolete warning. Please also get rid of the pragma warnings in Set-AzureRmApplicationGatewaySslCertificate.

Reply:
okay